### PR TITLE
Update how we launch subprocesses

### PR DESCRIPTION
**Pull request by lresende**
Developer's Certificate of Origin 1.1

       By making a contribution to this project, I certify that:

       (a) The contribution was created in whole or in part by me and I
           have the right to submit it under the Apache License 2.0; or

       (b) The contribution is based upon previous work that, to the best
           of my knowledge, is covered under an appropriate open source
           license and I have the right under that license to submit that
           work with modifications, whether created in whole or in part
           by me, under the same open source license (unless I am
           permitted to submit under a different license), as indicated
           in the file; or

       (c) The contribution was provided directly to me by some other
           person who certified (a), (b) or (c) and I have not modified
           it.

       (d) I understand and agree that this project and the contribution
           are public and that a record of the contribution (including all
           personal information I submit with it, including my sign-off) is
           maintained indefinitely and may be redistributed consistent with
           this project or the open source license(s) involved.

### CODE REVIEW

File: etc/docker-scripts/bootstrapper.py [hunk 1]
```diff
@@ -239,7 +233,7 @@ def execute(self) -> None:
         print("Executing Python Script : {} ==> {}".format(python_script, python_script_output))
         try:
             with open(python_script_output, "w") as log_file:
-                subprocess.check_call(['python', python_script], stdout=log_file, stderr=subprocess.STDOUT)
+                subprocess.run(['python', python_script], stdout=log_file, stderr=subprocess.STDOUT)
```
Comment (kevin-bates) [inline etc/docker-scripts/bootstrapper.py:236]: Does this need `check=True`?

**Review by kevin-bates — COMMENTED** [review]
Besides the one comment about `check=True`, a negative test is failing because, I believe, its checking for the wrong exception since `papermill` is now called via the command line.  I suspect it needs to be updated to check for [`CalledProcessError`](https://docs.python.org/3/library/subprocess.html#subprocess.CalledProcessError) since that's what `subprocess.run()` raises on non-zero exit statuses (when `check=True`).

File: etc/docker-scripts/bootstrapper.py (hunk 1 above)
Comment (lresende) [reply etc/docker-scripts/bootstrapper.py]: I have added `check=True` but it didn't make much difference...

**Review by kevin-bates — APPROVED** [review]
LGTM - thank you.

**Review by akchinSTC — APPROVED** [review]

**Pull request merged**